src/modules/user/dto/profile-questions.dto.ts

```diff
@@ -0,0 +1,56 @@
+import { IsNumber, IsOptional, IsString } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export default class ProfileQuestionsDto {
+  @ApiProperty({ example: 'Part-Time' })
+  @IsOptional()
+  @IsString()
+  available_time?: string;
+
+  @ApiProperty({
+    example: 'I code for 12 years, 6 years of producation exp, I like reading',
+  })
+  @IsOptional()
+  @IsString()
+  description?: string;
+
+  @ApiProperty({ example: 12 })
+  @IsOptional()
+  @IsNumber()
+  hourly_rate?: number;
+
+  @ApiProperty({ example: 'Full-Stack Developer' })
+  @IsOptional()
+  @IsString()
+  position?: string;
+
+  @ApiProperty({ example: 'I studied computer sciency at MIT' })
+  @IsOptional()
+  @IsString()
+  education_descr?: string;
```

Comment:
it will not have for both education and for work, as you can have multiple items in one time

Reply:
fixed. here are 3 new swaggers, instead of one
![image](https://user-images.githubusercontent.com/95277626/202782437-f36817be-9220-4009-bc82-09877eb52b68.png)
![image](https://user-images.githubusercontent.com/95277626/202782474-4fb15585-50b9-46bc-b119-1b86d7a5c586.png)
![image](https://user-images.githubusercontent.com/95277626/202782516-fb1b0c44-041f-48a2-b7e7-43fe0ce341e6.png)